motionit/src/main/java/com/back/motionit/global/error/code/CommentErrorCode.java

```diff
@@ -0,0 +1,18 @@
+package com.back.motionit.global.error.code;
+
+import org.springframework.http.HttpStatus;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+@Getter
+@RequiredArgsConstructor
+public enum CommentErrorCode implements ErrorCode {
+
+	WRONG_ACCESS(HttpStatus.FORBIDDEN, "U-101", "본인이 작성한 댓글이 아닙니다."),
```

Comment:
```suggestion
	WRONG_ACCESS(HttpStatus.FORBIDDEN, "M?-101", "본인이 작성한 댓글이 아닙니다."),
```
CommentErrorCode에는 Comment관련 응답형식으로 작성해주세요.